maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+            "frequency_penalty",
+        ]
+        for key in param_keys:
+            if key in kwargs and kwargs[key] is not None:
+                model_params[key] = kwargs[key]
+        for key, value in kwargs.items():
+            if key not in model_params and value is not None:
+                model_params[key] = value
+        return model_params
+
+    @staticmethod
+    def _message_content(message: Any) -> str:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `message`

(ANN401)